src/Tizen.Multimedia.Vision/MediaVision/RoiTrackingConfiguration.cs

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using InteropRoi = Interop.MediaVision.RoiTracker;
+
+namespace Tizen.Multimedia.Vision
+{
+    /// <summary>
+    /// Represents a configuration of <see cref="RoiTracker"/> instances.
+    /// </summary>
+    /// <feature>http://tizen.org/feature/vision.roi_tracking</feature>
+    /// <exception cref="NotSupportedException">The required features are not supported.</exception>
+    /// <since_tizen> 10 </since_tizen>
+    public class RoiTrackingConfiguration : EngineConfiguration
+    {
+        private IntPtr _handle;
+        private bool _disposed;
+        private const string KeyTrackerType = "MV_ROI_TRACKER_TYPE";
+
+        internal bool IsApplied { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RoiTrackingConfiguration"/> class.
+        /// </summary>
+        /// <rema
+        /// <exception cref="NotSupportedException">The feature is not supported.</exception>
+        /// <since_tizen> 10 </since_tizen>
+        public RoiTrackingConfiguration() : base("roi_tracking")
+        {
+            InteropRoi.Create(out _handle).Validate("Failed to create roi tracking configuration");
+        }
+
+        internal IntPtr GetRoiConfigHandle()
+        {
+            return _handle;
+        }
+
+        /// <summary>
+        /// Gets or sets the type of ROI tracker.<br/>
+        /// Default value is <see cref="RoiTrackerType.Balance"/>.
+        /// </summary>
+        /// <exception cref="ArgumentException"><paramref name="value"/> is not valid.</exception>
+        /// <since_tizen> 10 </since_tizen>
+        public RoiTrackerType ModelType
```

Comment:
Is there specific reason that tracker type enum name is 'ModelType' ?
Please let me suggestion.
In my opinion, 'TrackerType' is more straight-forward.
Because tracker is submodule of mediavision,it makes user confused that tracker uses model file for tracking. But tracker only uses algorithm for tracking not model.

So please let me know your opinion.

Reply:
I agree with you. I'll change it to `TrackerType`.

Reply:
Yes, thank you!